src/app/component/CategoryPagination/CategoryPagination.style.scss

```diff
@@ -0,0 +1,47 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+:root {
+    --color-pagination-link-text: var(--color-text-black);
+    --color-pagination-link-active-text: var(--link-color);
+}
+
+.CategoryPagination {
+    display: flex;
+    list-style: none;
+    justify-content: center;
+
+    &-ListItem {
+        line-height: 1.8;
+
+        &::before {
+            content: '';
+        }
+
+        @include mobile {
+            &:first-child {
+                padding-left: 0;
+            }
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        a {
```

Comment:
Please use class I asked you to add to a link above to style it. Add additional class in BEM style in order to style active state.